x-pack/plugins/search_assistant/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+require('@kbn/babel-register').install();
```

Comment:
Please remove this file, if we need specific linting rules (which we don't for now), we need to do that via the root of Kibana. We also should not be disabling the licensing header rule.